src/UI/Denominate/index.tsx

```diff
@@ -91,8 +94,14 @@ const DenominateComponent = (props: DenominateType) => {
     : denominateValid(props, props.egldLabel || '');
 };
 
+/**
+ * @param props.egldLabel  if not provided, will fallback on **EGLD**
+ */
 export const Denominate = (props: DenominateType) => {
-  const egldLabel = props.egldLabel || getEgldLabel();
+  const egldLabel =
+    props.egldLabel ||
+    fallbackNetworkConfigurations[EnvironmentsEnum.mainnet].egldLabel; // EGLD
```

Comment:
I wonder if we should extract this into a simple constant and use it inside configs, like we do with chainIds

Reply:
it's only this place we are using it like this. We can extract it to a constant if there will be one more place like this